src/FixPluginTypesSerialization/Patchers/IsFileCreated.cs

```diff
@@ -46,8 +46,18 @@ internal static void Dispose()
 
         private static unsafe bool OnIsFileCreated(IntPtr str)
         {
-            var assemblyString = UseRightStructs.GetStruct<IAbsolutePathString>(str);
-            var actualString = assemblyString.ToStringAnsi();
+            string actualString;
+
+            if (UseRightStructs.IsFileCreatedRelative)
```

Comment:
I don't get that if else statement, unless i'm missing something it's why we made the UseRightStructs abstraction in the first place, which pick the right struct to use while still using the same method calls through dynamic dispatch

Reply:
The parameter passed to `IsFileCreated` in this Unity version does not seem to be anything related to AbsolutePathString, so because I had to introduce a new struct, there needs to be something that can tell IsFileCreated to choose between the AbsolutePathString or the other struct. I did not want to create a new AbsolutePathString as I believe that struct itself might not have changed.

Reply:
The `ConvertSeparatorsToPlatform` still requires the use of the "current" AbsolutePathString (from v2021.v1), so that means if I added a new AbsolutePathString for v2022.v3.v62, it would break the other one.

Reply:
Only other thing I can think of is adding something like an `IIsFileCreatedParam` abstraction, that'd at least get rid of the if shenanigans

Reply:
Looks good to me, thanks!